sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### Bugs Fixed
 
 ### Other Changes
+- Renamed `enableSupportLogging` to `enableUnsafeSupportLogging`
```

Comment:
We should also mention, 
Added `BrowserCustomizationOptions` to `InteractiveBrowserCredentialBuilder`.